config/config.md

```diff
@@ -314,7 +314,7 @@
 
 | Key | Type | Default | Descriptions |
 | --- | -----| ------- | ----------- |
-| `data_home` | String | `./greptimedb_data/metasrv/` | The working home directory. |
+| `data_home` | String | `./greptimedb_data` | The working home directory. |
```

Comment:
Is this a breaking change if users are using the default configuration?

Reply:
It's not a breaking change.

In the original implementation, it uses the following code to create a data home for metasrv:

```rust
let metasrv_home = options.data_home.to_string();
```

This PR change to(`METASRV_DATA_DIR` is `metasrv`):

```rust
        let metasrv_home = Path::new(&options.data_home)
            .join(METASRV_DATA_DIR)
            .to_string_lossy()
            .to_string();

```

The final config is the same.

Reply:
After some thought, it's not breaking change for default configs. However, it could be a breaking change for the scenarios that specify `data_home`.

Reply:
What does this directory store?

Reply:
> What does this directory store?

Appears to contain only log files and temporary data like telemetry UUIDs.